README.md

```diff
@@ -1,150 +1,119 @@
 # workshop-template
 
 This repository is [Software Carpentry](http://software-carpentry.org)'s
-template for creating websites for workshops.  Do *not* fork this
-repository directly on GitHub.  Instead, follow the instructions below
-to create a website repository for your workshop (and possibly a
-second repository for your learners to use in your Git lessons).
-
-> If you are teaching Git, you should create a separate repository for
-> learners to use in that lesson.  You should not have them use the
-> workshop website repository because:
+template for creating websites for workshops.
+Do *not* fork this repository directly on GitHub.
+Instead, please follow the instructions below
+to create a website repository for your workshop.
+Please also read
+[the notes on customizing your website](CUSTOMIZATION.md) and the [FAQ](FAQ.md).
+If you're interested in knowing more about why we do things the way we do,
+please check out the [design notes](DESIGN.md).
+
+> If you are teaching Git,
```

Comment:
Perhaps this entire paragraph on teaching Git could be removed. It could be a recommendation in the Git lesson templates or alternatively later in these instructions after the user understands how to create a respository.